sample-app/src/components/InputDropdown.tsx

```diff
@@ -80,9 +87,31 @@ export default function InputDropdown({
     setScreenReaderKey(0);
   }
 
+  const screenReaderPhrases: string[] = [];
+  if (options.length < 1) {
+    const phrase = processTranslation({
+      phrase: `0 autocomplete option found.`,
+      pluralForm: `0 autocomplete options found.`,
+      count: 0
+    });
+    screenReaderPhrases.push(phrase);
+  } else {
+    options.forEach(section => {
+      const optionInfo = section.label? `${section.results.length} ${section.label}` : `${section.results.length}`;
+      const phrase = processTranslation({
+        phrase: `${optionInfo} autocomplete option found.`,
+        pluralForm: `${optionInfo} autocomplete options found.`,
+        count: section.results.length
+      });
+      screenReaderPhrases.push(phrase);
+    });
+  }
+  const screenReaderText = screenReaderPhrases.join(' ');
+
   function handleDocumentClick(evt: MouseEvent) {
     const target = evt.target as HTMLElement;
-    if (!target.isSameNode(inputRef.current)) {
+    if (!(target.isSameNode(inputRef.current) || target.classList.contains(cssClasses.optionContainer)
```

Comment:
The cssClasses object may contain a list of classes, so classList.contains wouldn't work in that scenario. Can we check in some other way that doesn't rely on the classList?

Reply:
The other way I thought of would involve useRef and forwardRef, but I think there would need to be a reference for each section label